ext/bcmath/libbcmath/src/recmul.c

```diff
@@ -111,10 +111,10 @@ static uint64_t bc_parse_chunk_chars(const char *str)
 #endif
 
 /*
- * Converts BCD to uint, going backwards from pointer n by the number of
+ * Converts BCD to int, going backwards from pointer n by the number of
```

Comment:
`convert to BC_INT_T` maybe?

Reply:
Fixed it
https://github.com/php/php-src/pull/14447/commits/c70de26ac37470b658dd6ec4926474a9764d63eb